frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectImportPortlet.java

```diff
@@ -85,7 +87,7 @@ private RemoteCredentials getRemoteCredentialsFromSession(PortletSession session
                 .setServerUrl(dbUrl);
     }
 
-    private void putRemoteCredentialsIntoSession(PortletSession session, RemoteCredentials remoteCredentials) {
+    private void putRemoteCredentialsAndProjectNameIntoSession(PortletSession session, RemoteCredentials remoteCredentials) {
```

Comment:
Why did you rename `putRemoteCredentialsIntoSession`?
In this method I can not see any put command for the project name.

Reply:
My bad, I reverted this.